h2o-algos/src/main/java/hex/schemas/GLMV3.java

```diff
@@ -64,7 +64,7 @@ public static final class GLMParametersV3 extends ModelParametersSchemaV3<GLMPar
             "link",
             "rand_link", // link function for random components, array
             "startval",  // initial starting values for fixed and randomized coefficients, double array
-            "calc_like", // HGLM, true will return likelhood function value
+            "calc_like", // true will return likelihood function value
             "HGLM",  // boolean: true - enabled HGLM, false - normal GLM
```

Comment:
Please change comment for calc_like here:

If not HGLM, true will return likelihood function value.  

Then in the parameter definition part, please make the change as well here around line 215 of the code in GLMV3.java
![image](https://github.com/h2oai/h2o-3/assets/7231712/e92855fc-2d43-4b95-a6ce-fdfcd9d6cb9a)

Reply:
For the line 67: This parameter is used by both HGLM and non-HGLM, so I just removed the limitation on the 67th line. Doe is make sense? In any case it controls the calculation of loglikelihood.

For the line 215: Thank you for pointing it out, I missed this comment. I would do it the same way as on the line 67 - "if true, will return likelihood function value." Is it OK for you?